ACadSharp.Tests/ACadSharp.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>net48;net5.0;net6.0</TargetFrameworks>
```

Comment:
I would suggest to change this in another branch so we don't break the scope for this task.

Reply:
Agreed. This was done mostly for #94 anyway. Not sure why I included that here. It ended up complicating things on the CI pipeline anyway.